contracts/0.8.25/vaults/VaultFactory.sol

```diff
@@ -73,7 +73,7 @@ contract VaultFactory {
         );
 
         // transfer ownership of the vault back to the delegation
-        vault.attachVaultHubAndDepositor();
+        vault.authorizeLidoVaultHub();
```

Comment:
the comment here is in the wrong place which is not in sync with the upstream version :eyes:

Reply:
fixed